### DISCUSSION TITLE
Fastest Proxy

### DISCUSSION THREAD

**Discussion by NoahHodgson** [scholarly-python-package/scholarly#439] [Q&A]
Hi there, trying to build something with fast-api and this library. Obviously the free proxy from the readme is going to be slower. Any recommendations on different proxies to use?

Thanks!

**Comment by arunkannawadi** [comment]
ScraperAPI has worked really well, until they jacked up their prices and made it prohibitively expensive for us to test our development. There are plans to use SerpAPI, but it may be another few weeks before a version that supports it.

**Reply by NoahHodgson** [reply]
Thank you I'll be looking forward to that then :) I can get by not using some of the functions until then.

**Reply by NoahHodgson** [reply]
Hey there. Wondering if this happened yet? Thanks!

**Reply by arunkannawadi** [reply]
SerpAPI works differently from the others, so we need to refactor our code base first. End of the year seems like a reasonable timescale by which you can expect this feature.
